public/js/main.js

```diff
@@ -15,6 +15,12 @@ var acelaya = {
 
         $form.submit(function (e) {
             e.preventDefault();
+
+            // remove .alert block when it exists
+            if ($(".container .alert").length) {
+                $(".container .alert").remove();
+            }
```

Comment:
Yes, but what I mean is that jQuery does not throw an error if provided selector is not found. It just does nothing, so there is no need to check that the element exists.